src/reflist.cpp

```diff
@@ -162,7 +162,10 @@ void RefList::generatePage()
       if (item->scope->name() != "<globalScope>")
       {
         doc += "\\_setscope ";
-        doc += item->scope->name();
+	if (item->scope->name().right(3)==" -p")
```

Comment:
I think we should find the cause of the space rather that dealing with it, because the -p is added only internally to avoid naming conflicts (class and protocol with the same name is allowed).

Reply:
I do agree, couldn't find it at first glance and the routine (as indicated in the question) that added the space is a quite often used routine. So I was a bit reluctant to do it. I'll have another look at it.

Reply:
I already found the problem and fixed it as part of this pull request.

Reply:
OK thanks.